COPYRIGHT.txt

```diff
@@ -1,5 +1,5 @@
 Copyright (c) 2021-2024 MLCommons
 
-The cTuning foundation and OctoML donated this project to MLCommons to benefit everyone.
+Grigori Fursin, the cTuning foundation and OctoML donated this project to MLCommons to benefit everyone.
```

Comment:
We should agree on the history and keep it stable - it doesn't look good if it keeps on changing. Technically cm4mlops and all the code inside it started within MLCommons.

Reply:
Parts of the code comes from older versions of CK and CM and from the https://github.com/mlcommons/ck where it was originally resided. We have to reflect that to avoid legal problems in the future.

Reply:
I originally started developing CM4MLOps as an integral part of CM while reusing some parts of CK and CK4MLOps that I developed at cTuning so it has to have a common history and license.

Reply:
https://octo.ai/blog/octoml-joins-the-community-effort-to-democratize-mlperf-inference-benchmarking - CM and CM4MLOps was a part of CK project at that time.

Reply:
The OctoAI blog has nothing to do with CM as it is talking about results taken using CK. Anyway I do not want to argue over these things. Since none of the projects are new, I request you to finalize the historical attributions and contributions and and keep them constant. This can help others decide in using the project. If anyone has a concern that the history of a project can change in future no one will be happy to contribute to it. Or else, let git history take care of them automatically as done in most other MLCommons repositories.

Reply:
Sure! Agree! Thank you!

But we still have a problem with the conflict in .github/workflows/test-mlperf-inference-resnet50.yml . 

May I ask you to help resolve it and merge the PR, please? 

Thanks a lot for your help - very appreciated!